packages/page-accounts/src/Sidebar/AddressSection.tsx

```diff
@@ -19,6 +21,8 @@ interface Props {
 
 function AddressSection ({ accountIndex, defaultValue, editingName, flags, onChange, value }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
+  const [isCopyShown, toggle] = useToggle();
```

Comment:
Consider renaming `toggle` to `toggleIsCopyShown`

Reply:
done